docs/querying/query-context-reference.md

```diff
@@ -33,6 +34,14 @@ HTTP POST API, or as properties to the JDBC connection.
 Note that setting query context will override both the default value and the runtime properties value in the format of
 `druid.query.default.context.{property_key}` (if set). 
 
+This reference contains all available context parameters organized by their scope: 
+
+- **General parameters**: Apply to all query types and control execution behavior, resource limits, and caching
+- **Query type-specific parameters**: Available only for certain query types like TopN, Timeseries, or GroupBy
+- **Vectorization parameters**: Control vectorized query execution for supported query types
+
+For instructions on how the different approaches for set query context in your queries, see [Set the Query Context](./set-query-context.md).
```

Comment:
```suggestion
To learn how to set the query context, see [Set query context](./set-query-context.md).
```

Reply:
Suggest re-reviewing all references to the new doc to ensure
1. correct title
2. correct capitalization